packages/grid/_modules_/grid/components/GridCell.tsx

```diff
@@ -52,6 +50,7 @@ export const GridCell: React.FC<GridCellProps> = React.memo((props) => {
   const valueToRender = formattedValue || value;
   const cellRef = React.useRef<HTMLDivElement>(null);
   const apiRef = React.useContext(GridApiContext);
+  const { align = 'left', field } = column;
```

Comment:
Why do you need the column now?

Reply:
I need it to call `onColItemDragOver` later in this file. Previously I just had the field and align from the column as props.

Reply:
in #1201 we will remove this method of apiRef as they don't really make sense. 
I think #1201 should come before this changes.
This method start an action instead of letting you subscribe to an event.

You can refactor by passing the field and retrieving the column directly in the handler using apiRef.